src/crimson/os/seastore/cache.h

```diff
@@ -584,15 +591,63 @@ class Cache {
    */
   CachedExtent::list dirty;
 
-  using src_ext_t = std::pair<Transaction::src_t, extent_types_t>;
+  struct query_counters_t {
+    uint64_t access;
+    uint64_t hit;
+  };
+
+  struct effort_t {
+    uint64_t extents = 0;
+    uint64_t bytes = 0;
```

Comment:
it'd be difficult to tell what `extents` and `bytes` is for in this context. the same applies to `effort_t`.

Reply:
The effort is counted by the number of extents and the bytes of these extents. So would `num_extents` and `bytes_extents` better?

And is `trans_effort_t` a better name?

Reply:
i mean, probably we could add a comment explaining that this represents the total number of extents and the total bytes of them involved in transactions, they are categorized by different sets of extents involved by transactions. like read_set , retireset, ...

Reply:
Oh, sure. I'll add them.

Reply:
done.